bot/tests/administration/test_report_issue.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from bot.tests.base_test import BaseTest
+
+
+class TestReportCommand(BaseTest):
```

Comment:
Znowu dość generyczna uwaga, ale jak chcemy prawdziwe testy e2e, to powinieneś się wpinać do bazy i assertować że komendy z whitelistą czy inne reporty faktycznie wpływają na stan bazy

A poza tym przyda ci się jakaś kontrola nad bazą do setupowania/teardownowania środowiska testowego raczej więc dwie pieczenie na jednym ogniu